### ISSUE TITLE
Help component popup width does not format correctly

### ISSUE THREAD

**Issue by abierbaum** [Teradata/covalent#1474]
### Bug Report

When using the experimental help component and opening a draggable help window, it is not possible to adjust the width of the dialog.  See work around below.

##### Example

As per the documentation example, consider calling as:

```
this.ref = this.draggableHelpWindowDialogService.open({
   items: [{url: 'https://github.com/Teradata/covalent/blob/develop/README.md'}],
   dialogConfig: { width: '500px'}
});
```

When doing this, the help width and dialog content width does not adjust.  

#### What is the expected behavior?

The window should adjust to the size passed

#### Which version of Angular and Material, and which browser and OS does this issue affect?

Used with latest experimental code from head of the nightly branch

##### Details and workaround

The issue is partially because of a fixed width set in the help-window component style.

See: https://github.com/Teradata/covalent/blob/develop/src/platform/experimental/help/help-window/help-window.component.scss#L7

As a work around in our current application I am setting the following at the global style scope:

```
td-help-window {
   display: block !important;
   width: 100%;
   height: 100%;

   td-help {
      width: 100% !important;
   }
}
```

I didn't see the best way to fix this more generically in the code base and cover any corner cases.  Posting here in case anyone else runs up against this issue.

**Comment by christianmemije** [comment]
@abierbaum Thank you for reporting this bug. We are currently working on making some improvements to this component, so we will address this too soon.

**Comment by christianmemije** [comment]
Fixed as part of #1478
